raster-foundry/raster-foundry
# Use correct query builder for campaign tasks

## Overview

This PR makes sure that we use the right query-builder for fetching campaign tasks.

### Checklist


## Testing Instructions

- Have a campaign with at least one image and some tasks
- `GET /api/campaigns/{id}/tasks` and verify that you get tasks back
- filter by some status to ensure filters are being applied correctly ex: `GET /api/campaigns/{id}/tasks?status=FLAGGED`
